src/wms/src/main/java/org/geoserver/wms/map/AbstractOpenLayersMapOutputFormat.java

```diff
@@ -292,7 +293,7 @@ private List<String> styleNames(WMSMapContent mapContent) {
             MapLayerInfo info = mapContent.getRequest().getLayers().get(0);
             result = info.getOtherStyleNames();
         }
-        return result;
+        return result.stream().map(StringEscapeUtils::escapeHtml4).collect(Collectors.toList());
```

Comment:
Why are the other parameters escaped in the template, while this one is escaped in Java code?

Reply:
I see, the explanation makes sense, thanks!